### ISSUE TITLE
WCAG: Tieto lukemattomasta viestistä ei käy ilmi ruudunlukijalle

### ISSUE THREAD

**Issue by ninakurki** [otavanopisto/muikku#6788]
**Vakava ongelma**
1.3.1 Informaatio ja suhteet (A): Ei täyty

Päänavigaation Viestin-linkissä osoitetaan numerolla ja punaisella ympyrällä tieto, että käyttäjällä on yksi lukematon viesti. Tieto ei kuitenkaan ole saavutettavissa ruudunlukijalla.

**Korjausehdotus:** Ruudunlukija ei todennäköisesti lue punaisen ympyrän numeroa kohdassa käytetyn ::before pseudoelementin vuoksi. Jos pseudoelementeillä asetellaan ei-koristeellista sisältöä, ruudunlukijat eivät välttämättä enää pysty lukemaan tällaista tietoa. [Lisätietoa pseudoelementtien saavutettavuudesta W3C:n sivuilla](https://www.w3.org/WAI/WCAG21/Techniques/failures/F87).
